rubrics/tool/src/main/frontend/webcomponents/js/sakai-rubrics-manager.js

```diff
@@ -104,6 +111,27 @@ class SakaiRubricsManager extends RubricsElement {
       icon.removeClass("fa-chevron-down").addClass("fa-chevron-right");
     }
   }
+
+  filterRubrics() {
+    var rubricsList = this.querySelector('sakai-rubrics-list');
+    var sharedRubricsList = this.querySelector('sakai-rubrics-shared-list');
+    var searchInput = document.getElementById('rubrics-search-bar');
+    var searchInputValue = searchInput.value.toLowerCase();
+
+    $([rubricsList, sharedRubricsList]).find('.rubric-item').each(function () {
+      var rubricItem = $(this);
+      rubricItem.removeClass('hidden');
+      var rubricTitle = $(this).find('.rubric-name').text();
+      var rubricAuthor = $(this).find('sakai-rubric-creator-name').text();
+      var rubricSite = $(this).find('sakai-rubric-site-title').text();
+      if (!rubricAuthor.toLowerCase().includes(searchInputValue) &&
```

Comment:
You don't need to wrap the element in a jquery object repeatedly. You're already creating the rubricsList.